__tests__/paddle.ts

```diff
@@ -0,0 +1,174 @@
+import { DataSource } from 'typeorm';
+import createOrGetConnection from '../src/db';
+import { saveFixtures } from './helpers';
+import { User } from '../src/entity';
+import { plusUsersFixture, usersFixture } from './fixture';
+import { EventName, SubscriptionCreatedEvent } from '@paddle/paddle-node-sdk';
+import {
+  PaddleCustomData,
+  updateUserSubscription,
+} from '../src/routes/webhooks/paddle';
+import { isPlusMember, SubscriptionCycles } from '../src/paddle';
+import { FastifyInstance } from 'fastify';
+import appFunc from '../src';
+
+let app: FastifyInstance;
+let con: DataSource;
+
+beforeAll(async () => {
+  con = await createOrGetConnection();
+  app = await appFunc();
+  return app.ready();
+});
+
+beforeEach(async () => {
+  await saveFixtures(con, User, [...usersFixture, ...plusUsersFixture]);
```

Comment:
can you also add prefixes as I mentioned in cron PR?

Reply:
Added ✔️